### PR TITLE
bootstrap added

### PR DESCRIPTION
**Pull request by bmw95**

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -11,3 +11,6 @@
  *= require_self
  *= require_tree .
  */
+
+@import "bootstrap-sprockets";
+@import "bootstrap";
```
Comment (lasley) [inline app/assets/stylesheets/application.css]: FYI you should have one blank line at the end of every file. This is called a line terminator, and is part of the POSIX file spec - http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -11,6 +11,8 @@
 // about supported directives.
 //
 //= require jquery
+//= require bootstrap-sprockets
 //= require jquery_ujs
 //= require turbolinks
 //= require_tree .
+
```
Comment (lasley) [inline app/assets/javascripts/application.js]: But this is too many blank lines :stuck_out_tongue:

**Comment by bmw95** [comment]
Sure I'll get that fixed. Rubocop was pretty useful once I got it set up properly, but there were too many issues with the gem so I decided to get rid of it.

**Comment by lasley** [comment]
Too many issues with the gem? What was wrong with it?

**Comment by bmw95** [comment]
https://github.com/Homebrew/homebrew/issues/48105

I'd keep getting this issue. It happened no matter whether I specified the newer cop spec or older spec, or none in the config file. Tried version 34.2 and 37.2 and still kept getting the same response in the running test.

**Comment by lasley** [comment]
Weird - my team is using 0.37.2 without issues. Oh well, not a biggie - Travis is the main one that we need, particularly with this being a fairly simple app. I recommend taking this ticket when it comes up in the team project so you can give it another shot ;)

**Pull request merged**